tasks/cache_config.yml

```diff
@@ -25,16 +25,22 @@
 
 - name: Ensure LSC files present
   copy:
-    dest: "{{ voms_dir }}/{{ item.name}}/{{ item.voms.hostname }}.lsc"
+    dest: "{{ voms_dir }}/{{ item.name}}/{{ item.hostname }}.lsc"
     content: |
-      {{ item.voms.DN}}
-      {{ item.voms.CA_DN}}
-  loop: "{{ vo_voms | json_query('data[*]')}}"
+      {{ item.DN}}
+      {{ item.CA_DN}}
+  loop:
+    "{{ vo_voms | json_query('data[*]') | subelements('voms') | to_json |
+    from_json | json_query('[*].{name: [0].name, hostname: [1].hostname, DN:
+    [1].DN, CA_DN: [1].CA_DN}') }}"
 
 - name: Ensure vomses files present
   copy:
-    dest: "{{ vomses_dir }}/{{ item.name }}-{{ item.voms.hostname }}"
+    dest: "{{ vomses_dir }}/{{ item.name }}-{{ item.hostname }}"
     content: >
-                        "{{ item.name }}" "{{ item.voms.hostname}}" "{{ item.voms.port }}"
-                        "{{ item.voms.DN }}" "{{ item.name }}"
-  loop: "{{ vo_voms | json_query('data[*]')}}"
+      "{{ item.name }}" "{{ item.hostname}}" "{{ item.port }}" "{{ item.DN }}"
+      "{{ item.name }}"
+  loop:
+    "{{ vo_voms | json_query('data[*]') | subelements('voms') | to_json |
```

Comment:
🤯 let's assume you know what you are doing

Reply:
Thanks stackoverfow ;)
This one was a real pain to debug, probably too complex, but at least I was able to get voms-proxy from the dteam VO that got configured with this ino...